gramhopper/dict_enum.py

```diff
@@ -2,14 +2,16 @@
 
 
 class DictEnumMeta(EnumMeta):
-    def __getitem__(self, key):
+    """Metaclass for DictEnum"""
+
+    def __getitem__(cls, key):
         key_path = key.split('.')
         item = super().__getitem__(key_path.pop(0)).value
 
-        if len(key_path) > 0:
+        if key_path:
```

Comment:
I never really liked this syntax. I know it works, and it's that more _python way_ of doing things, but it's much less readable than the previous `if len()>0`.
Anyway, if fuhrerlint insists on doing that, let it be.